connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/EmailRouteBuilder.java

```diff
@@ -284,6 +330,31 @@ public void configureRoute() throws Exception {
             .end();
     }
 
+    /**
+     * Computes the cache key.
+     * @return a simple expression with the "${orgId}-adminsOnly=${adminsOnly}"
+     * format.
+     */
+    protected Expression computeCacheKey() {
+        return simpleF(
+            "${exchangeProperty.%s}-adminsOnly=${exchangeProperty.%s.adminsOnly}",
+            ORG_ID,
+            ExchangeProperty.CURRENT_RECIPIENT_SETTINGS
+        );
+    }
+
+    /**
+     * Sets a common configuration for all the Caffeine caches.
+     */
+    protected void configureCaffeineCache() {
+        final CaffeineCacheComponent caffeineCacheComponent = this.getCamelContext().getComponent("caffeine-cache", CaffeineCacheComponent.class);
+        final CaffeineConfiguration caffeineConfiguration = caffeineCacheComponent.getConfiguration();
+
+        // We explicitly set the eviction type as time based, since the default
+        // value is size based: https://camel.apache.org/components/4.0.x/caffeine-cache-component.html.
+        caffeineConfiguration.setEvictionType(EvictionType.TIME_BASED);
```

Comment:
How can we configure the expiration delay of the cache entries? We'll need that in this PR.

Reply:
Thanks, that's what I was looking for.